catalyst-gateway/tests/schemathesis_tests/Earthfile

```diff
@@ -2,46 +2,36 @@ VERSION 0.8
 
 package-schemathesis:
     FROM python:3.12-alpine3.19
-
-    ARG tag="latest"
-    ARG max_examples=1000
     # TODO: https://github.com/input-output-hk/catalyst-voices/issues/465
-    ARG max_response_time=3000
-    ARG wait_for_schema=15
-    ARG workers=2
-    ARG schema_version=30
-    ARG openapi_spec
-
     RUN apk add --no-cache gcc musl-dev
     RUN python -m pip install schemathesis==3.27.1
     RUN mkdir /results
-    COPY ./hooks/hooks.py .    
-
+    COPY ./hooks/hooks.py .
     VOLUME /results
+    ARG openapi_spec
     ENTRYPOINT export SCHEMATHESIS_HOOKS=hooks \
                     && st run --checks all $openapi_spec \
-                    --workers=$workers \
-                    --wait-for-schema=$wait_for_schema \
-                    --max-response-time=$max_response_time \
-                    --hypothesis-max-examples=$max_examples \
+                    --workers=2 \
+                    --wait-for-schema=15 \
+                    --max-response-time=3000 \
+                    --hypothesis-max-examples=1000 \
                     --data-generation-method=all \
                     --skip-deprecated-operations \
-                    --force-schema-version=$schema_version \
+                    --force-schema-version=30 \
                     --show-trace \
                     --force-color  \
                     --junit-xml=/results/junit-report.xml \
                     --cassette-path=/results/cassette.yaml
-
+    ARG tag="latest"
     SAVE IMAGE schemathesis:$tag
 
 # test-fuzzer-api - Fuzzy test cat-gateway using openapi specs
 test-fuzzer-api:
     FROM earthly/dind:alpine-3.19
     RUN apk update && apk add iptables-legacy # workaround for https://github.com/earthly/earthly/issues/3784
     RUN apk add yq zstd
-    ARG OPENAPI_SPEC="http://127.0.0.1:3030/docs/cat-gateway.json"
-
     COPY schemathesis-docker-compose.yml .
+    ARG OPENAPI_SPEC="http://127.0.0.1:3030/docs/cat-gateway.json"
```

Comment:
Do we really need this argument ?
Seems it is not going to be really specified because it's a `test` target.
Maybe we can just remove it

Reply:
We can use a `LET` instead `ARG` here.